client/components/marked-lines/index.js

```diff
@@ -0,0 +1,86 @@
+/** @format */
```

Comment:
Why did you use .js for this file extension when other components use .jsx?
https://github.com/Automattic/wp-calypso/blob/08244a217f3eb728e747aa6e07cfbe898f86508c/client/components/button/index.jsx

Reply:
good question! oversight on my part… renamed in d6bc8824b3